nrich-validation/src/test/java/net/croz/nrich/validation/constraint/validator/ValidOibValidatorTest.java

```diff
@@ -29,11 +30,9 @@ void shouldSkipValidationOfNullOrEmptyValue(String oib) {
         assertThat(result).isTrue();
     }
 
-    @Test
-    void shouldReturnFalseForInvalidOib() {
-        // given
-        String oib = "11111111110";
-
+    @ValueSource(strings = { "aaa", "11111111110" })
```

Comment:
Maybe use something meaningful instead of `"aaa"`. Maybe something like `"invalidOib"`.